doc/tutorial/conv_arithmetic.txt

```diff
@@ -0,0 +1,180 @@
+.. _conv_arithmetic:
+
+======================
+Convolution arithmetic
+======================
+
+The relationship between a convolution operation's input shape, kernel size,
+stride, padding and its output shape can be confusing at times.
+
+This tutorial aims at providing an intuitive understanding of how the different
+convolution parameters influence the shape of the output it produces.
+
+Refresher: discrete convolution
+===============================
+
+.. todo::
+
+    WRITEME
```

Comment:
When you will write this say something like "The convolution operation can be performed on one or more axes, also known as 1-D, 2-D, ..., n-D convolution", so that we can then refer to that in the rest of the tutorial.